File: BasicCourseOOP/1. CursoPOOUber/Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted=[]
    seatsMaterial=[]

    def __init__(self,license,driver,typeCarAccepted,seatsMaterial):
        super(UberVan,self).__init__(license,driver)
        self.typeCarAccepted=typeCarAccepted
        self.seatsMaterial=seatsMaterial
        

File: BasicCourseOOP/1. CursoPOOUber/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car{
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    
    public UberVan(String license, Account driver, 
    Map<String, Map<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted=typeCarAccepted;
        this.seatsMaterial=seatsMaterial;
    }

    public UberVan(String license, Account driver){
        super(license, driver);
    }

    @Override
    public void setPassenger(Integer passenger) {
        if(passenger==6){
            super.passengers=passenger;
        }else{
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }

}
